src/main/java/org/folio/service/budget/BudgetService.java

```diff
@@ -147,8 +145,9 @@ public Future<Budget> getBudgetByFundIdAndFiscalYearId(String fiscalYearId, Stri
           handleFailure(promise, reply);
         } else if (reply.result()
           .isEmpty()) {
-          logger.error(BUDGET_NOT_FOUND_FOR_TRANSACTION);
-          promise.fail(new HttpException(Response.Status.BAD_REQUEST.getStatusCode(), BUDGET_NOT_FOUND_FOR_TRANSACTION));
+          logger.error("Budget not found for fundId: {} and fiscalYearId: {}", fundId, fiscalYearId);
+          promise.fail(new HttpException(Response.Status.NOT_FOUND.getStatusCode(),
+            JsonObject.mapFrom(BUDGET_NOT_FOUND_FOR_TRANSACTION.toError()).encodePrettily()));
```

Comment:
BUDGET_NOT_FOUND_FOR_TRANSACTION - it is just string message, can we omit using JsonObject?

Reply:
it actually not string anymore, in new case I introduce error code mod-finance-storage also, it is now giving Error object we should use JsonObject I guess
![image](https://user-images.githubusercontent.com/113523904/211818280-8fbf3f19-2e87-4101-8209-b4aeda0cf923.png)

Reply:
I seem this approach also used in all other places, so I am ok with that